README.md

```diff
@@ -53,7 +53,8 @@ For more information, see the [wiki](https://github.com/Azure/azure-storage-fuse
 	* [OPTIONAL] **--file-cache-timeout-in-seconds=120** : Blobs will be cached in the temp folder for this many seconds. 120 seconds by default. During this time, blobfuse will not check whether the file is up to date or not.
 	* [OPTIONAL] **--log-level=LOG_WARNING** : Enables logs written to syslog. Set to LOG_WARNING by default. Allowed values are LOG_OFF|LOG_CRIT|LOG_ERR|LOG_WARNING|LOG_INFO|LOG_DEBUG
 	* [OPTIONAL] **--use-attr-cache=true|false** : Enables attributes of a blob being cached. False by default. (Only available in blobfuse 1.1.0 or above)
-
+    * [OPTIONAL] **--use-adls=true|false** : Enables Azure DataLake storage
```

Comment:
Somewhere in this README or maybe in the wiki we should put in some warning about using ADLS on a storage account without Hierarchical Namespace enabled. Also a link to ADLS (https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction) and how that works and what we don't support might help as well. (This doesn't have to be in this PR)

Reply:
Will update readme with this link and will specify there that wrong account type can lead of unpredictable behavior. From code side we do not have any way to know the type so we can not throw any error or warning if wrong type is supplied.